templates/terraform/constants/pubsub_topic.go.erb

```diff
@@ -0,0 +1,17 @@
+<%- # the license inside this block applies to this file
+	# Copyright 2020 Google Inc.
+	# Licensed under the Apache License, Version 2.0 (the "License");
+	# you may not use this file except in compliance with the License.
+	# You may obtain a copy of the License at
+	#
+	#     http://www.apache.org/licenses/LICENSE-2.0
+	#
+	# Unless required by applicable law or agreed to in writing, software
+	# distributed under the License is distributed on an "AS IS" BASIS,
+	# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	# See the License for the specific language governing permissions and
+	# limitations under the License.
+-%>
+func pubsubPollWaiter(res map[string]interface{}) PollWaiter {
```

Comment:
Why does pollWaiter take the response? It looks like neither implementation uses it.

Reply:
for the state-checking one, in case you need the response from the last request to create/update